Day2/RibbonPaperCalculator.py

```diff
@@ -0,0 +1,36 @@
+from math import pi
+
+class RibbonPaperCalculator():
+    
+    total_ribbon_required = 0
```

Comment:
total redundans sot ha ribbonCalculator a class neve akkor meg a ribbon is megkerdojelezheto h kell e

Reply:
mindig bezavar h egy nev tul specifikus vagy nem eleg absztrakt de a mostaniaknal en ugy erzem h a legtobb nem tul specifikus hanem redundans es attol hosszu

beleraksz szavakat ami ha logikusan vegiggondolod nem ad semmi extra infot